junifer/storage/sqlite.py

```diff
@@ -445,7 +445,7 @@ def store_matrix(
 
         Parameters
         ----------
-        data : dict
+        data : numpy.ndarray
```

Comment:
`ArrayLike`?

Reply:
does not work with ArrayLike because we use `.shape`